app/gui/src/components/TabView.vue

```diff
@@ -83,11 +91,19 @@ const onSignOut = () => {
           @update:selected="$event && setPage(project.id)"
         >
           <SvgIcon v-if="readyProjects.has(project.id)" name="graph_editor" />
-          <LoadingSpinner v-else :size="16" />
+          <GrowingSpinner v-else :state="loadingProjectSpinnerState(project)" :size="16" />
```

Comment:
I think `state` is a bit abstract, and in the context of a spinner sounds like it would include progress.

Reply:
The term is taken from react implementation: https://github.com/enso-org/enso/blob/wip/farmaazon/tab-view-fixes/app/gui/src/dashboard/components/Spinner.tsx#L9. Frankly, I cannot make any better name ATM. Any ideas?